### PR TITLE
Added video of IO venue

### PR DESCRIPTION
**Pull request by DarshilBhatt**

### CODE REVIEW

File: _posts/2018-07-18-io-extended.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ date: 2018-07-18
 categories: ["Events"]
 tags: ["Google IO", "IO 2018 Extended"]
 author: Darshil Bhatt
+assets: "/assets/2018-07/Google IO 2018 Extended"
```
Comment (manan-jadhav) [inline _posts/2018-07-18-io-extended.md:8]: Dont need this really

File: _posts/2018-07-18-io-extended.md [hunk 2]
```diff
@@ -16,7 +17,7 @@ Before live streaming started, some members of community delivered the talks on
 * Google Maps
 * Google Cloud Platform
 
-We enjoyed seeing the video of Google IO venue which was made by the two co-organizers of GDG Baroda who were at Shoreline.That was just an awesome! Finally,the live streaming started with the rocking keynote by Sundar Pichai. It was a feeling like as we are actually attending the event at Shoreline itself.We were just enjoying the keynote in the anticipation of products that are coming up.Sundar Pichai & other Google leads were speaking about the coming products of Google.This time in IO, the big announcements in the Google technologies were:-
+We enjoyed seeing the [video](https://photos.app.goo.gl/XsnBjAXxPFttZwfz7) of Google IO venue which was made by the two co-organizers of GDG Baroda who were at Shoreline.That was just an awesome! Finally,the live streaming started with the rocking keynote by Sundar Pichai. It was a feeling like as we are actually attending the event at Shoreline itself.We were just enjoying the keynote in the anticipation of products that are coming up.Sundar Pichai & other Google leads were speaking about the coming products of Google.This time in IO, the big announcements in the Google technologies were:-
```
Comment (manan-jadhav) [inline _posts/2018-07-18-io-extended.md:20]: How about embedding the video instead of linking to it?

**Review by manan-jadhav — CHANGES_REQUESTED** [review]
Please add images of the extended event

**Comment by DarshilBhatt** [comment]
Removed assets column, added images & video embedded.

**Pull request closed without merging**